app/labor/bufferizer.rb

```diff
@@ -21,21 +21,22 @@ def satellite_tweet!
       if text.length < 250 && SiteConfig.twitter_hashtag
         text = text.gsub(" #{SiteConfig.twitter_hashtag}", " #{SiteConfig.twitter_hashtag} ##{tag.name}")
       end
-      BufferUpdate.buff!(@article.id, text, tag.buffer_profile_id_code, "twitter", tag.id, @admin_id)
+      BufferUpdate.buff!(@article.id, text, tag.buffer_profile_id_code, { tag_id: tag.id, admin_id: @admin_id })
     end
     @article.update(last_buffered: Time.current)
   end
 
   def main_tweet!
-    BufferUpdate.buff!(@article.id, twitter_buffer_text, ApplicationConfig["BUFFER_TWITTER_ID"], "twitter", nil,
-                       @admin_id)
+    BufferUpdate.buff!(@article.id, twitter_buffer_text, ApplicationConfig["BUFFER_TWITTER_ID"],
+                       { admin_id: @admin_id })
     @article.update(last_buffered: Time.current)
   end
 
   def facebook_post!
-    BufferUpdate.buff!(@article.id, fb_buffer_text, ApplicationConfig["BUFFER_FACEBOOK_ID"], "facebook", nil, @admin_id)
-    BufferUpdate.buff!(@article.id, fb_buffer_text + social_tags, ApplicationConfig["BUFFER_LINKEDIN_ID"], "linkedin",
-                       nil, @admin_id)
+    BufferUpdate.buff!(@article.id, fb_buffer_text, ApplicationConfig["BUFFER_FACEBOOK_ID"],
```

Comment:
Following my last suggestion, you could create on `ApplicationConfig` an accessor to return that ID using the `social_service_name` and extract it directly inside de `BufferUpdate.buff!`